tests/core/test_solve.py

```diff
@@ -3076,9 +3092,7 @@ def test_freeze_deps_1(tmpdir):
             (
                 "channel-2::six-1.7.3-py34_0",
                 "channel-2::python-3.4.5-0",
-                # LIBMAMBA ADJUSTMENT
-                # libmamba doesn't remove xz in this solve
-                *(() if context.solver == "libmamba" else ("channel-2::xz-5.2.3-0",)),
+                "channel-2::xz-5.2.3-0",
```

Comment:
v2 fixed this solver difference. We can still bring it back with [this change](https://github.com/conda/conda-libmamba-solver/pull/457/files#diff-e6794455e7cacff948cf95915e290bc1f742360246a7a96f809b7627bbc379c4R524-R527), though.